sys/arm64/acpica/acpi_machdep.c

```diff
@@ -66,16 +66,16 @@ static void *
 map_table(vm_paddr_t pa, const char *sig)
 {
 	ACPI_TABLE_HEADER *header;
-	vm_offset_t length;
+	vm_size_t length;
```

Comment:
NB: Upstreamed this pedantic change in https://cgit.freebsd.org/src/commit/?id=52fba9a943cca518d2978f7588e99a35da37e5f2